requirements.in

```diff
@@ -11,4 +11,6 @@ djangorestframework
 django-braces
 drf-yasg
 odata-request-parser  # Deze library wordt door ons beheerd: https://github.com/Amsterdam/odata_request_parser
-django-http-exceptions
\ No newline at end of file
+django-http-exceptions
+certifi>=2022.12.07  # Without this pin a lower version of certifi is installed, which poses a security vulnerability
```

Comment:
Dit is nu nodig maar is natuurlijk binnen de kortste keren outdated. Ik vraag me af hoe vaak we straks dit soort dingen handmatig in moeten gaan stellen om trivy blij te houden. Heb jij al gedachten over gehad?